inscriptions/ISic003454.xml

```diff
@@ -194,14 +199,25 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="#MB">
+                <ab>
+                    <lb n="1"/><g ref="#crux"/> Ένθάδε
+                    <lb n="2"/>κὲτε Ἰωά
```

Comment:
woudl be good to tag the regularised form of κειται here, using `<choice><orig>κὲτε</orig><reg>κεῖται</reg></choice>`